waves_gateway/controller/flask_rest_controller.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, flask: Any, gateway_controller: GatewayController, logger: Lo
         self._serializer = attempt_list_serializer
         self._public_configuration = public_configuration
         self._public_configuration_serializer = public_configuration_serializer
+        self._key = "CHANGE_THIS_KEY"
```

Comment:
The api key must be overhanded to the Gateway class and then be injected here.
It is not possible to define this in the code. This would require the Gateway developer to change the source code of the framework which is a no-go...

Reply:
done